autogpt_platform/frontend/src/components/NavBar.tsx

```diff
@@ -40,64 +43,58 @@ export async function NavBar() {
           <SheetContent side="left">
             <nav className="grid gap-6 text-lg font-medium">
               <Link
-                href="/"
-                className="flex flex-row gap-2 text-muted-foreground hover:text-foreground"
+                href="/marketplace"
+                className="mt-4 flex flex-row items-center gap-2 text-muted-foreground hover:text-foreground"
               >
-                <IconSquareActivity /> Monitor
+                <LuShoppingCart /> Marketplace
               </Link>
               <Link
-                href="/build"
-                className="flex flex-row gap-2 text-muted-foreground hover:text-foreground"
+                href="/"
+                className="flex flex-row items-center gap-2 text-muted-foreground hover:text-foreground"
               >
-                <IconWorkFlow /> Build
+                <LuLaptop /> Monitor
               </Link>
               <Link
-                href="/marketplace"
-                className="flex flex-row gap-2 text-muted-foreground hover:text-foreground"
+                href="/build"
+                className="flex flex-row items-center gap-2 text-muted-foreground hover:text-foreground"
               >
-                <IconPackage2 /> Marketplace
+                <BsBoxes /> Build
               </Link>
             </nav>
           </SheetContent>
         </Sheet>
-        <nav className="hidden md:flex md:flex-row md:items-center md:gap-5 lg:gap-6">
+        <nav className="hidden md:flex md:flex-row md:items-center md:gap-7 lg:gap-8">
+          <div className="flex h-10 w-20 flex-1 flex-row items-center justify-center gap-2">
+            <a href="https://agpt.co/">
+              <Image
+                src="/AUTOgpt_Logo_dark.png"
+                alt="AutoGPT Logo"
+                width={100}
+                height={40}
```

Comment:
Changing height from 20 to 40, won't affect anything since it's maintaining its ratio.